packages/ui-drilldown/src/Drilldown/README.md

```diff
@@ -2,11 +2,336 @@
+- secondary information blocks can be displayed both in-line with the option, or below the option with a dedicated text and color style;
+- the component can be rendered both in-line and in a popover.
+
+```js
+---
+example: true
+---
+<Drilldown rootPageId='root' width="20rem" maxHeight="30rem">
+  <Drilldown.Page id="root" renderTitle='Groceries'>
+    <Drilldown.Option id="Produce" subPageId="Produce">
+      Produce
+    </Drilldown.Option>
+    <Drilldown.Option id="Grains and Bread" subPageId="Grains and Bread">
+      Grains and Bread
+    </Drilldown.Option>
+  </Drilldown.Page>
+
+  <Drilldown.Page id="Produce" renderTitle='Produce'>
+    <Drilldown.Option id="Fruits" subPageId="Fruits">
+      Fruits
+    </Drilldown.Option>
+    <Drilldown.Option id="Vegtables" subPageId="Vegtables">
+      Vegtables
+    </Drilldown.Option>
+  </Drilldown.Page>
+
+  <Drilldown.Page id="Grains and Bread" renderTitle='Grains and Bread'>
+    {['Pasta', 'Rice', 'Bread', 'Flour', 'Cereal', 'Oats'].map(
+      item => <Drilldown.Option id={item} key={item}>{item}</Drilldown.Option>
+    )}
+  </Drilldown.Page>
+
+  <Drilldown.Page id="Fruits" renderTitle='Fruits'>
+    {['Apple', 'Orange', 'Cherry', 'Grapefruit', 'Mango', 'Banana', 'Strawberry'].map(
+      item => <Drilldown.Option id={item} key={item}>{item}</Drilldown.Option>
+    )}
+  </Drilldown.Page>
+
+  <Drilldown.Page id="Vegtables" renderTitle='Vegtables'>
+    {['Tomato', 'Cucumber', 'Eggplant', 'Lettuce', 'Garlic', 'Onion', 'Corn', 'Carrot', 'Bell pepper'].map(
+      item => <Drilldown.Option id={item} key={item}>{item}</Drilldown.Option>
+    )}
+  </Drilldown.Page>
+</Drilldown>
+```
+
+### Options
+
+`Drilldown.Option` is the main child component of Drilldown. The content can be a ReactNode or a function returning a ReactNode. The function has an object as its parameter, containing the option's `id`, `variant` and `isSelected` state.
+
+> Note: Drilldown is based on the [Options](#Options) component, so the Drilldown.Options are rendered as `Options.Item` components under the hood. This is why the `variant` parameter has the values of Options.Item's `variant` prop.
+
+```js
+---
+example: true
+---
+<Drilldown rootPageId='root' width="20rem" maxHeight="30rem">
+  <Drilldown.Page id="root">
+    <Drilldown.Option id="option1">
+      Option
+    </Drilldown.Option>
+    <Drilldown.Option id="option2">
+      {(props) => `Option ${props.variant === 'highlighted'
+        ? 'is highlighted'
+        : 'is not highlighted'}`}
+    </Drilldown.Option>
+    <Drilldown.Option id="option3">
+      <Pill>Pill</Pill> Option
+    </Drilldown.Option>
+    <Drilldown.Option id="option4" disabled>
+      Option
+    </Drilldown.Option>
+  </Drilldown.Page>
+</Drilldown>
+```
+
+Options can be links too. If the `href` prop is set, the option renders as an `<a>` element.
```

Comment:
Shouldn't we render links with an underline? For me its confusing that sometimes I'm navigated away from a page by clicking something, sometimes I'm not and there is no visual difference. Also its a [WCAG requirement](https://webaim.org/techniques/hypertext/link_text#appearance)

Reply:
Menu items can be links too, and those have no underline either. It has to be consistent with Menu.

Reply:
OK, then lets postphone this discussion